docs/non-deterministic-error.md

```diff
@@ -287,11 +287,11 @@ func (wc *workflowEnvironmentImpl) GenerateSequence() int32 {
 
 THE_ID_WITH_ERROR is the ActivityID/TimerID of activities/timers of decision runs into non-deterministic error during your replay with current code. When you pause the replay thread in the fmt.Println("PAUSE HERE IN DEBUG MODE") , trace back in the stack you will see the position of your workflow code that run into non-deterministic error.
 
-Currently this debugging experience is not very ideal. [This proposal will help](https://github.com/uber/cadence/issues/2801). 
+Currently this debugging experience is not very ideal. [This proposal will help](https://github.com/uber/temporal/issues/2801). 
 
 ## Automatic history replay and non-deterministic error detection
 
-We have ideas on how non-deterministic errors can be detected automatically and safe rollout can be achieved. See [this issue](https://github.com/uber/cadence/issues/2547).
+We have ideas on how non-deterministic errors can be detected automatically and safe rollout can be achieved. See [this issue](https://github.com/uber/temporal/issues/2547).

```

Comment:
Pointing to invalid issues now

Reply:
Yeah.  We need to do that as a separate PR.